src/backend/joanie/core/permissions.py

```diff
@@ -30,3 +32,15 @@ def has_object_permission(self, request, view, obj):
         """Check permission for a given object."""
         abilities = obj.get_abilities(request.user)
         return abilities.get("sign_contracts", False)
+
+
+class HasAPIKey(permissions.BasePermission):
+    """Permission class to grant access to our remote endpoints API."""
+
+    def has_permission(self, request, view):
+        """
+        Check if the token is set in the request and known in our settings
+        `JOANIE_AUTHORIZED_API_TOKENS`.
+        """
+        token = request.headers.get("Authorization")
```

Comment:
The value is not `Bearer xxxxxx` ? You have to remove the `bearer` string from it.

Reply:
I have an updated version of this pull request. I have added to my permission class that we need the "Bearer" prefix before our token string. Bearer is now used everywhere for the remote endpoints :heavy_check_mark:

Reply:
Here is an updated version of the permission class: 
```python
class HasAPIKey(permissions.BasePermission):
    """Permission class to grant access to our remote endpoints API."""

    def has_permission(self, request, view):
        """
        Check if a valid authorization token is present in the request headers.

        This method verifies whether the token is included in the 'Authorization' header
        and follows the 'Bearer' scheme. It then checks if the token exists in the list
        of authorized tokens specified by the `JOANIE_AUTHORIZED_API_TOKENS` variable.
        """
        authorization_header = request.headers.get("Authorization")
        if not authorization_header:
            return False

        token_prefix, token = authorization_header.split()
        if token_prefix != 'Bearer':
            return False

        return token in settings.JOANIE_AUTHORIZED_API_TOKENS
```

Reply:
For this first implementation, we decide to simply pass the token without prefix.
Are there drawbacks ?